src/domain.module.ts

```diff
@@ -71,6 +72,7 @@ import { FiatCodesValidator } from '@/domain/prices/fiat-codes.validator';
     ConfigApiModule,
     ExchangeApiModule,
     PricesApiModule,
+    AlertsApiModule,
```

Comment:
nitpick: I'd leave this in lexicographical order.

(It's already not in the correct order though... so we can either tackle it here or in a different PR).

Reply:
I've moved this particular module to the correct alphabetical position in 36844eeae590fc1a6dd9d1ec0e87d8645e760b2d. I agree that we should tackle the rest in a separate PR.